frontend/src/pages/journal/components/journal-sidebar/styles.module.scss

```diff
@@ -1,30 +1,52 @@
 .info {
   display: flex;
-  justify-content: center;
   align-items: center;
 }
 
-.journal-entry-list {
+.chat-number {
```

Comment:
chat naming in journal sidebar component?

Reply:
@ossavemu  I am more curious why it was added to my PR to begin with?

Reply:
i guess these changes can be should be deleted ?

Reply:
Ye, I'll delete them in a bit